src/index.js

```diff
@@ -1,3 +1,19 @@
-import { getNearestShops } from './app.js';
+import getNearestShops from "./app.js";
 
-getNearestShops();
+// Get argument list from command line
+const position = process.argv.slice(2);
+
+let input = {
```

Comment:
Can you tell us the why of using let, and when to use `let` vs `const`?

Reply:
When I see this, I immediately realized my mistake here :) I used `let` where I could use `const` here. `const` should be used when the variable value does not need a change. `let` should be used when the variable value needs to be changed later. So, the `input` variable here should have been defined with `const`.